content/zh/docs/concepts/_index.md

```diff
@@ -72,10 +72,10 @@ The basic Kubernetes objects include:
 
 基本的 Kubernetes 对象包括：
 
-* [Pod](/docs/concepts/workloads/pods/pod-overview/)
-* [Service](/docs/concepts/services-networking/service/)
-* [Volume](/docs/concepts/storage/volumes/)
```

Comment:
we  need to update this?
it  seems not good.
/cc @tengqm

Reply:
@snowei
Thanks for your contribution

Reply:
We still have to do it today.

Reply:
done . get  it .